### PR TITLE
Rearranged examples with the current functionality

### PR DESCRIPTION
**Pull request by jeffreypaul15**
Both 
- [x] #26 
- [x] #29 

need to be merged first and then a rebase.

image

### CODE REVIEW

File: changelog/30.trivial.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Adds an example tutorial which deals with overplotting of the brightest pixel with :meth:`~sunkit_pyvista.plotter.SunpyPlotter.plot_coordinates`.
```
Comment (nabobalis) [inline changelog/30.trivial.rst:1]:
```suggestion
Added an example which deals with overplotting of the brightest pixel with :meth:`~sunkit_pyvista.plotter.SunpyPlotter.plot_coordinates`.
```

File: examples/skip_plot_brighest_pixel.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+"""
+=======================================
```
Comment (nabobalis) [inline examples/skip_plot_brighest_pixel.py:2]: Was this meant to be a new example?

Comment (jeffreypaul15) [reply examples/skip_plot_brighest_pixel.py:2]: Yeah, I thought this would be better as a separate example

Comment (nabobalis) [reply examples/skip_plot_brighest_pixel.py:2]: I don't think there is enough different here to warrant a new example?

File: examples/skip_plot_functionality.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+m = Map(AIA_171_IMAGE)
+
+# Start by creating a plotter
+plotter = SunpyPlotter()
+# Plot a map
+plotter.plot_map(m, clip_interval=(1, 99.9)*u.percent)
+# Add an arrow to show the solar rotation axis
+plotter.plot_solar_axis()
+
+# We can also plot an arbitrary point by passing a single coordinate to
```
Comment (nabobalis) [inline examples/skip_plot_functionality.py:32]:
```suggestion
###############################################################################
# We can also plot an arbitrary point by passing a single coordinate to
```

File: examples/skip_plot_functionality.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+"""
+=====================
+Adding extra features
+=====================
+
+sunkit-pyvista also extends features from `sunpy`, producing
+them in 3D.
```
Comment (nabobalis) [inline examples/skip_plot_functionality.py:7]:
```suggestion
sunkit-pyvista also extends features from `sunpy`, producing them in 3D.
```
What features?

File: examples/skip_plot_functionality.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+from sunpy.data.sample import AIA_171_IMAGE
+from sunpy.map import Map
+
+from sunkit_pyvista import SunpyPlotter
+
+###############################################################################
+# We will firstly use an AIA 171 image from the sunpy sample data as the base image.
+m = Map(AIA_171_IMAGE)
+
+# Start by creating a plotter
```
Comment (nabobalis) [inline examples/skip_plot_functionality.py:25]:
```suggestion
###############################################################################
# Start by creating a plotter
```

File: examples/skip_plot_functionality.py (hunk 4 above)
Comment (jeffreypaul15) [reply examples/skip_plot_functionality.py:7]: The examples listed out here? Should they be called something else?

Comment (nabobalis) [reply examples/skip_plot_functionality.py]: It needs to explain what these features are.

File: examples/skip_plot_functionality.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
+"""
+=====================
+Adding extra features
+=====================
+
+sunkit-pyvista also extends ``draw_quadrangle``, and ``plot_coord`` from `sunpy`, producing them in 3D.
```
Comment (nabobalis) [inline examples/skip_plot_functionality.py:6]: Can we link these?

File: examples/skip_plot_functionality.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
+"""
+=====================
+Adding extra features
```
Comment (nabobalis) [inline examples/skip_plot_functionality.py:3]: I think this title needs changing.

Comment (jeffreypaul15) [reply examples/skip_plot_functionality.py:3]: Does "Extending functionality from sunpy" sound better?

Comment (nabobalis) [reply examples/skip_plot_functionality.py]: Yeah that works.

File: examples/skip_plot_functionality.py (hunk 6 above)
Comment (jeffreypaul15) [reply examples/skip_plot_functionality.py:6]: Sure

**Review by nabobalis — APPROVED** [review]

**Pull request merged**